opennms-alarms/api/src/main/java/org/opennms/netmgt/alarmd/api/NorthboundAlarm.java

```diff
@@ -280,7 +284,11 @@ public static x733AlarmType get(int id) {
 	
     private final Integer m_id;
     private final String m_uei;
-	private Integer m_nodeId;
```

Comment:
The ElasticSearch NorthBounder (merged in 18.0.0) relies on the nodeId.

See https://github.com/OpenNMS/opennms/blob/develop/features/elasticsearch/event-forwarder/src/main/java/org/opennms/features/elasticsearch/eventforwarder/internal/NodeCache.java for details.